script.js

```diff
@@ -0,0 +1,339 @@
+  const targetMini = currentMini.previousElementSibling;
+  const index = slides.findIndex((i) => i === prevImage);
+  movingSlide(track, currentImage, prevImage);
+  movingMinis(currentMini, targetMini);
+  movingTitle(titles, currentTitle, prevTitle);
+  hideButton(index);
+});
+
+prevProject.classList.add('btn-none');
+prevProject.addEventListener('click', () => {
+  const currentImage = track.querySelector('.active');
+  const prevImage = currentImage.previousElementSibling;
+  const currentText = text.querySelector('.active-paragraph');
+  const prevText = currentText.previousElementSibling;
+  const currentTitle = titles.querySelector('.active-title');
+  const prevTitle = currentTitle.previousElementSibling;
+  const currentMini = miniSlides.querySelector('.active-mini-slide');
+  const targetMini = currentMini.previousElementSibling;
+  const index = slides.findIndex((i) => i === prevImage);
+  movingSlide(track, currentImage, prevImage);
+  movingText(text, currentText, prevText);
+  movingMinis(currentMini, targetMini);
+  movingTitle(titles, currentTitle, prevTitle);
+  hideProject(index);
+});
+
+nextButton.addEventListener('click', () => {
+  const currentImage = track.querySelector('.active');
+  const nextImage = currentImage.nextElementSibling;
+  const currentText = text.querySelector('.active-paragraph');
+  const nextText = currentText.nextElementSibling;
+  const currentTitle = titles.querySelector('.active-title');
+  const nextTitle = currentTitle.nextElementSibling;
+  const currentMini = miniSlides.querySelector('.active-mini-slide');
+  const targetMini = currentMini.nextElementSibling;
+  const index = slides.findIndex((i) => i === nextImage);
+  movingSlide(track, currentImage, nextImage);
+  movingText(text, currentText, nextText);
+  movingMinis(currentMini, targetMini);
+  movingTitle(titles, currentTitle, nextTitle);
+  hideButton(index);
+});
+
+nextProject.addEventListener('click', () => {
+  const currentImage = track.querySelector('.active');
+  const nextImage = currentImage.nextElementSibling;
+  const currentText = text.querySelector('.active-paragraph');
+  const nextText = currentText.nextElementSibling;
+  const currentTitle = titles.querySelector('.active-title');
+  const nextTitle = currentTitle.nextElementSibling;
+  const currentMini = miniSlides.querySelector('.active-mini-slide');
+  const targetMini = currentMini.nextElementSibling;
+  const index = slides.findIndex((i) => i === nextImage);
+  movingSlide(track, currentImage, nextImage);
+  movingText(text, currentText, nextText);
+  movingTitle(titles, currentTitle, nextTitle);
+  movingMinis(currentMini, targetMini);
+  hideProject(index);
+});
+
+form.addEventListener('submit', (e) => {
+  const message = [];
+
+  if (formName.value === '') {
+    message.push('Please enter your name');
+    e.preventDefault();
+  }
+  if (formEmail.value !== formEmail.value.toLowerCase() || formEmail.value.toUpperCase()) {
```

Comment:
- [ ] Well done 👏🏻 you did great work but I can see that your form is not working because your condition is not correct or it might be possible that you forget to delete the `condition` after checking the **formEmail** to lowerCase. Even if I tried to submit the email all in small letters but it didn't work. Kindly follow the below suggestion to make your form work correctly.

```suggestion
  if (formEmail.value !== formEmail.value.toLowerCase()) {
```